lib/constants/color_constants.dart

```diff
@@ -0,0 +1,4 @@
+import 'dart:ui';
+
+const Color kPrimaryColor = const Color(0xFF0F1236);
+const Color kSecondaryColor = const Color(0xFF45CAB9);
```

Comment:
Most flutter apps that I've checked out also had `k` as a prefix to global color constants.

Reply:
We should nail down a style guide that we should use moving forward. For example, Effective Dart does not use prefix letters (https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters). Also my understanding is that `k` prefixing and other Hungarian Notations are used for the Dart project but not for projects and applications developed with Dart

Reply:
Agreed! Here's the page the I saw regarding this: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k
I definitely don't mind doing either way, we can talk about it at our next standup!
P.S. the `k` prefix looks wack anyways :sob: